### PR TITLE
bugfix: right link for transfer

### LINKED ISSUE #3753: Our transfer function generates weird link that doesnt work
### What happened?

I’ve used transfer function

### Please reproduce in steps

It generates this link

 https://undefined.subscan.io/extrinsic/0xbad28463be97b63443de249dada01683a5ee7b48a509c1c0b0e51ae93ab707a1

Instead of this one

https://kusama.subscan.io/extrinsic/14031877-15

### Expected Behavior

/

### What browsers are you seeing the problem on?

Firefox

### At which address did you encounter bug?

kodadot.xyz

### Are you logged in?

Yes

### Which wallet you are using?

PolkadotJS

### At which chain did you encounter bug?

RMRK

### Screenshots

image

### Relevant log output

_No response_

### Payment link for reward

https://kodadot.xyz/transfer/?target=%3CMy_Kusama_Address_check_https://github.com/kodadot/nft-gallery/blob/main/CONTRIBUTING.md#creating-your-ksm-address%3E

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by zhengow**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

### What's new?

- [x] PR closes #3753
- [ ] Requires deployment <rubick/magick_issue>
- [ ] <brief_description_of_what_I've_added>

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

### Optional

- [x] I've tested it at </transfer>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=EgCzZLpVe7zg7RHh79qPyC8UrykVkA22YY2oj7sWMZf15Ws)

### Community participation

- [ ] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

https://user-images.githubusercontent.com/38389586/185198838-97fe1a4f-9dd8-417b-9d91-4eb1b3d44784.mp4

### CODE REVIEW

File: utils/config/chain.config.ts [hunk 1]
```diff
@@ -24,7 +24,7 @@ const DEFAULT_CHAIN_PROPERTIES: ChainProperties = toChainProperty(
 
 const chainPropertyMap: Config<ChainProperties> = {
   rmrk: DEFAULT_CHAIN_PROPERTIES,
-  bsx: toChainProperty(10041, 12, 'BSX'),
+  bsx: toChainProperty(10041, 12, 'BSX', 'https://basilisk.subscan.io/'),
```
Comment (roiLeo) [inline utils/config/chain.config.ts:27]: not ready as we still are on bsx testnet
This show "false" link in history component

**Review by roiLeo — COMMENTED** [review]
I'm not sure about these changes, they could break things.

**Comment by JustLuuuu** [comment]
I still cannot see my transaction.
image

**Comment by zhengow** [comment]
> I still cannot see my transaction. image

Because currently we are in bsx testnet. 🤔

File: utils/config/chain.config.ts (hunk 1 above)
Comment (zhengow) [reply utils/config/chain.config.ts:27]: So maybe I still hide bsx link, only show ksm link, is this ok?

Comment (roiLeo) [reply utils/config/chain.config.ts:27]: yeah hide it for bsx for now until we go with mainnet
I think it was possible to see the transactions/blocks using polkdadotjs

Comment (zhengow) [reply utils/config/chain.config.ts:27]: What's the url? Could u pls give a demo, thanks🙂

Comment (roiLeo) [reply utils/config/chain.config.ts:27]: https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frpc-01.basilisk.hydradx.io#/explorer

with custom node endpoint

Comment (zhengow) [reply utils/config/chain.config.ts:27]: Thanks!
So do we still need to hide current bsx(testnet) or we use this endpoint to show something?

Comment (roiLeo) [reply utils/config/chain.config.ts:27]: hide bsx, I sense that polkadot js link will be hard to find/make

Comment (zhengow) [reply utils/config/chain.config.ts:27]: Ok

**Comment by zhengow** [comment]
> I'm not sure about these changes, they could break things.

Hide bsx link, pls check:)

**Review by roiLeo — APPROVED** [review]
Transfer BSX worked for me 

✅ code lgtm

**Comment by roiLeo** [comment]
Is this PR related with #3661? might breaks code

**Review by JustLuuuu — APPROVED** [review]
wfm

**Comment by yangwao** [comment]
> Is this PR related with #3661? might breaks code

I guess not related?

**Comment by roiLeo** [comment]
> I guess not related?

Parts of the code removed here are used in #3661

**Comment by zhengow** [comment]
> > I guess not related?
> 
> Parts of the code removed here are used in #3661

The function `urlBuilderBlockNumber` in `utils/explorerGuide.ts` is removed by commit [8abf8e](https://github.com/zhengow/nft-gallery/commit/8abf8efd3b5ca5399dc7bf510fd104ede8e2470e) 👀

**Comment by yangwao** [comment]
pay 50

**Comment by yangwao** [comment]
pay 50

**Comment by yangwao** [comment]
pay 50

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 49.75 USD/KSM ~ 1.005 $KSM
🧗 [EgCzZLpVe7zg7RHh79qPyC8UrykVkA22YY2oj7sWMZf15Ws](https://kusama.subscan.io/account/EgCzZLpVe7zg7RHh79qPyC8UrykVkA22YY2oj7sWMZf15Ws)
🔗 [0x93496f160b66fd11a1b081bf391368bcccf62a900faa304916e66080c4f468c2](https://kusama.subscan.io/extrinsic/0x93496f160b66fd11a1b081bf391368bcccf62a900faa304916e66080c4f468c2)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Comment by yangwao** [comment]
seems autocode <> github having timeouts 👀

**Pull request merged**